daemon/vpn/nordlynx/libtelio/libtelio.go

```diff
@@ -169,32 +176,46 @@ func New(prod bool, eventPath string, fwmark uint32,
 	events := make(chan state)
 	logLevel := teliogo.TELIOLOGINFO
 
-	cfg, err := handleTelioConfig(eventPath, deviceID, appVersion, prod, telioCfg)
-	if err != nil {
-		log.Println(internal.ErrorPrefix, "failed to get telio config: ", err)
-
-		defaultTelioConfig := &telioFeatures{}
-		defaultTelioConfig.Lana = &lanaConfig{
-			Prod:      prod,
-			EventPath: eventPath,
-		}
-		defaultTelioConfig.Direct = &directConfig{}
-		defaultTelioConfig.Nurse = &nurseConfig{
-			Fingerprint: deviceID,
+	var telioConfigString string
+	if val, ok := os.LookupEnv(TelioLocalConfigName); ok && strings.TrimSpace(val) != "" {
```

Comment:
What about moving this logic to remote config implementation instead?

Reply:
Well, essentially it's `local` config and not `remote`, but yes, it's still `config`...

Reply:
Refactored